spec/compiler/lexer/lexer_spec.cr

```diff
@@ -153,13 +153,17 @@ describe "Lexer" do
   it_lexes "\n", :NEWLINE
   it_lexes "\n\n\n", :NEWLINE
   it_lexes "_", :UNDERSCORE
-  it_lexes_keywords [:def, :if, :else, :elsif, :end, :true, :false, :class, :module, :include,
-                     :extend, :while, :until, :nil, :do, :yield, :return, :unless, :next, :break,
-                     :begin, :lib, :fun, :type, :struct, :union, :enum, :macro, :out, :require,
-                     :case, :when, :select, :then, :of, :abstract, :rescue, :ensure, :is_a?, :alias,
-                     :pointerof, :sizeof, :instance_sizeof, :offsetof, :as, :as?, :typeof, :for, :in,
-                     :with, :self, :super, :private, :protected, :asm, :uninitialized, :nil?,
-                     :annotation, :verbatim]
+  it_lexes_keywords :def, :if, :else, :elsif, :end, :true, :false, :class, :module, :include,
+    :extend, :while, :until, :nil, :do, :yield, :return, :unless, :next, :break,
+    :begin, :lib, :fun, :type, :struct, :union, :enum, :macro, :out, :require,
+    :case, :when, :select, :then, :of, :abstract, :rescue, :ensure, :alias,
+    :pointerof, :sizeof, :instance_sizeof, :offsetof, :as, :typeof, :for, :in,
+    :with, :self, :super, :private, :protected, :asm, :uninitialized,
+    :annotation, :verbatim
+  it_lexes "is_a?", :IDENT, Keyword::IS_A_QUESTION
+  it_lexes "as?", :IDENT, Keyword::AS_QUESTION
+  it_lexes "nil?", :IDENT, Keyword::NIL_QUESTION
+  it_lexes "responds_to?", :IDENT, Keyword::RESPONDS_TO_QUESTION
```

Comment:
Do these keywords actually need to be handled separately? I would expect them to work with `it_lexes_keywords`.